<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignUpView</class>
 <widget class="QWidget" name="SignUpView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>OctoVault VPN</string>
  </property>
  <widget class="QLabel" name="lblBackgroundImg">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>500</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>800</width>
     <height>500</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>800</width>
     <height>500</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../assets/assets.qrc">:/img/images/img_background.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblTitle">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>110</y>
     <width>200</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>26</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>26</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
color:#FFFFFF;
}</string>
   </property>
   <property name="text">
    <string>Get started!</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblSubTitle">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>140</y>
     <width>200</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>16</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>162</width>
     <height>16</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
color:#FACA7F;
}</string>
   </property>
   <property name="text">
    <string>Start your 7-days free trial</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSignIn">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>450</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>20</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <bold>false</bold>
     <underline>false</underline>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
color: #FFFFFF;
background-color: transparent;
}</string>
   </property>
   <property name="text">
    <string>Have an account? Log in!</string>
   </property>
  </widget>
  <widget class="QWidget" name="usernameHolder" native="true">
   <property name="geometry">
    <rect>
     <x>275</x>
     <y>180</y>
     <width>250</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>250</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>180</width>
     <height>30</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{
background:transparent;
border: 1px solid #36353a;
border-radius: 10px;
padding-left: 10px;
}

QWidget:hover{
background:transparent;
border: 1px solid #FACA7F;
border-radius: 10px;
padding-left: 10px;
}</string>
   </property>
   <widget class="QLineEdit" name="editEmail">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>200</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
background:transparent;
padding-left:0px;
color: #36353a;
border-color: transparent;
}

QLineEdit:hover{
background:transparent;
padding-left:0px;
color: #FACA7F;
border-color: transparent;
}</string>
    </property>
    <property name="placeholderText">
     <string>Email</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="passHolder" native="true">
   <property name="geometry">
    <rect>
     <x>275</x>
     <y>230</y>
     <width>250</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>250</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>250</width>
     <height>40</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{
background:transparent;
border: 1px solid #36353a;
border-radius: 10px;
padding-left: 10px;
}

QWidget:hover{
background:transparent;
border: 1px solid #FACA7F;
border-radius: 10px;
padding-left: 10px;
}</string>
   </property>
   <widget class="QPushButton" name="btnSHPass">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>30</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>30</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>30</width>
      <height>20</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-color: transparent;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normalon>:/img/images/icon_show.png</normalon>
     </iconset>
    </property>
   </widget>
   <widget class="QLineEdit" name="editPass">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>200</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
background:transparent;
padding-left:0px;
color: #36353a;
border-color: transparent;
}

QLineEdit:hover{
background:transparent;
padding-left:0px;
color: #FACA7F;
border-color: transparent;
}</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Password</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="confirmHolder" native="true">
   <property name="geometry">
    <rect>
     <x>275</x>
     <y>280</y>
     <width>250</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>250</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>250</width>
     <height>40</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{
background:transparent;
border: 1px solid #36353a;
border-radius: 10px;
padding-left: 10px;
}

QWidget:hover{
background:transparent;
border: 1px solid #FACA7F;
border-radius: 10px;
padding-left: 10px;
}</string>
   </property>
   <widget class="QPushButton" name="btnSHPConfirm">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>30</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>30</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>30</width>
      <height>20</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-color: transparent;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normalon>:/img/images/icon_show.png</normalon>
     </iconset>
    </property>
   </widget>
   <widget class="QLineEdit" name="editConfirm">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>200</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
background:transparent;
padding-left:0px;
color: #36353a;
border-color: transparent;
}

QLineEdit:hover{
background:transparent;
padding-left:0px;
color: #FACA7F;
border-color: transparent;
}</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Confirm Password</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnSignUp">
   <property name="geometry">
    <rect>
     <x>275</x>
     <y>410</y>
     <width>250</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background:#FACA7F;
border-radius: 15px;
color:#000000;
}</string>
   </property>
   <property name="text">
    <string>Sign Up</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblSpinner">
   <property name="geometry">
    <rect>
     <x>385</x>
     <y>360</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblLogo">
   <property name="geometry">
    <rect>
     <x>385</x>
     <y>50</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../assets/assets.qrc">:/img/images/logo.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblIntro">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>350</y>
     <width>200</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
color:#FFFFFF;
}</string>
   </property>
   <property name="text">
    <string>By signing up you agree to our</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblAndText">
   <property name="geometry">
    <rect>
     <x>385</x>
     <y>370</y>
     <width>30</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
color:#FFFFFF;
}</string>
   </property>
   <property name="text">
    <string>and</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnTerms">
   <property name="geometry">
    <rect>
     <x>295</x>
     <y>370</y>
     <width>90</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background:transparent;
color:#FACA7F;
}</string>
   </property>
   <property name="text">
    <string>Terms of Service</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPrivacy">
   <property name="geometry">
    <rect>
     <x>415</x>
     <y>370</y>
     <width>80</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background:transparent;
color:#FACA7F;
}</string>
   </property>
   <property name="text">
    <string>Privacy Policy</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblLogoTwo">
   <property name="geometry">
    <rect>
     <x>335</x>
     <y>50</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../assets/assets.qrc">:/img/images/logo_text.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnChangeLogo">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>50</y>
     <width>160</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background:transparent;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../assets/assets.qrc"/>
 </resources>
 <connections/>
</ui>
